logdna/resource_stream_exclusion.go

```diff
@@ -0,0 +1,185 @@
+package logdna
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type streamExclusion struct {
+	ID     string   `json:"id,omitempty"`
+	Title  string   `json:"title"`
+	Active bool     `json:"active"`
+	Apps   []string `json:"apps"`
+	Hosts  []string `json:"hosts"`
+	Query  string   `json:"query"`
+}
+
+var streamExclusionAtLeastOneOfFields = []string{"apps", "hosts", "query"}
+
+func resourceStreamExclusionCreate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	pc := m.(*providerConfig)
+	ex := streamExclusion{
+		Title:  d.Get("title").(string),
+		Active: d.Get("active").(bool),
+		Apps:   listToStrings(d.Get("apps").([]interface{})),
+		Hosts:  listToStrings(d.Get("hosts").([]interface{})),
+		Query:  d.Get("query").(string),
+	}
+
+	req := newRequestConfig(
+		pc,
+		"POST",
+		"/v1/config/stream/exclusions",
+		ex,
+	)
+
+	body, err := req.MakeRequest()
+	if err != nil {
+		return diag.FromErr(err)
+	}
+
+	c := streamExclusion{}
```

Comment:
Nit: similar point on consistency & readability, might be better to rename this variable to `created`